apps/intellij/src/main/kotlin/dev/nx/console/ide/ProjectGraphErrorProblemProvider.kt

```diff
@@ -0,0 +1,96 @@
+package dev.nx.console.ide
+
+import com.intellij.analysis.problemsView.FileProblem
+import com.intellij.analysis.problemsView.Problem
+import com.intellij.analysis.problemsView.ProblemsCollector
+import com.intellij.analysis.problemsView.ProblemsProvider
+import com.intellij.codeInsight.daemon.impl.HighlightInfo
+import com.intellij.codeInsight.daemon.impl.HighlightInfo.newHighlightInfo
+import com.intellij.codeInsight.daemon.impl.HighlightInfoType
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFileManager
+import com.intellij.workspaceModel.ide.getInstance
+import dev.nx.console.models.NxError
+import dev.nx.console.nxls.NxWorkspaceRefreshListener
+import dev.nx.console.nxls.NxlsService
+import dev.nx.console.utils.nxBasePath
+import java.io.File
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@Service(Service.Level.PROJECT)
+class ProjectGraphErrorProblemProvider(val project: Project, val cs: CoroutineScope) {
+    private val nxProblemsProvider = NxProblemsProvider(project)
+    private val nxlsService = NxlsService.getInstance(project)
+
+    private val problems = mutableListOf<Problem>()
+
+    fun init() {
+        setNxProblems()
+        with(project.messageBus.connect()) {
```

Comment:
Nit: Is this the same as

```
project.messageBus
  .connect()
  .subscribe(
    NxlsService.NX_WORKSPACE_REFRESH_TOPIC,
    NxWorkspaceRefreshListener { setNxProblems() }
  )
```

?

Reply:
Yeah it's the same. The `with` syntax is useful when subscribing to multiple topics but useless here.